SWLOR.Game.Server/Feature/DialogDefinition/RenameItemDialog.cs

```diff
@@ -0,0 +1,103 @@
+﻿using SWLOR.Game.Server.Core;
+using SWLOR.Game.Server.Core.NWNX;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.DialogService;
+using static SWLOR.Game.Server.Core.NWScript.NWScript;
+
+namespace SWLOR.Game.Server.Feature.DialogDefinition
+{
+    public class RenameItemDialog : DialogBase
```

Comment:
Went old school with this one, huh? ;)

I think I'd rather we make a small GUI for handling this instead of the old style conversation. It'll be more consistent.